lib/galaxy/tours/_impl.py

```diff
@@ -22,6 +22,11 @@ def load_tour_steps(contents_dict):
     #  Some of this can be done on the clientside.  Maybe even should?
     title_default = contents_dict.get("title_default")
     for step in contents_dict["steps"]:
+        # Remove attributes no longer used, so they are attempted to be
+        # validated.
+        if "backdrop" in step:
```

Comment:
Should we also validate this in your new script and provide a warning when people use it?

Reply:
I didn't particularly like it, but was backdrop useful?  We can also just reimplement this if desirable.

Reply:
Don't know to be honest.

Reply:
I would argue this PR didn't remove the functionality so if someone wants to restore it... they should do that on top this 😅 . Also it feels like no one is jumping to implement that so hopefully it isn't important it was lost. Wait for complaints before the UI team picks this up?

Reply:
@jmchilton Yep, wasn't saying this was dropping it or anything, I know where that happened --  was just asking if it was potentially useful to ressurect.  I'm fine with letting it fall off, though, if nobody complains.